client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -0,0 +1,181 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import property from 'lodash/utility/property';
+import config from 'config';
+import classNames from 'classnames';
+import analytics from 'analytics';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import ButtonGroup from 'components/button-group';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import SelectDropdown from 'components/select-dropdown';
+import DropdownItem from 'components/select-dropdown/item';
+import DropdownSeparator from 'components/select-dropdown/separator';
+import BulkSelect from 'components/bulk-select';
+import PluginsActions from 'lib/plugins/actions';
+
+export default React.createClass( {
+	displayName: 'Plugins-list-header',
+
+	propTypes: {
+		bulkManagement: React.PropTypes.bool,
+		sites: React.PropTypes.object.isRequired,
+		plugins: React.PropTypes.array.isRequired,
+		selected: React.PropTypes.array.isRequired,
+		isWpCom: React.PropTypes.bool,
+		pluginUpdateCount: React.PropTypes.number
+	},
+
+	onBrowserLinkClick() {
+		analytics.ga.recordEvent( 'Plugins', 'Clicked Add New Plugins' );
+	},
+
+	canAddNewPlugins() {
+		if ( config.isEnabled( 'manage/plugins/browser' ) ) {
```

Comment:
since the amount of properties this component receives is quite big already, I rather prefer to read this from inside the component than passing the prop... but I'm open to arguments against it :)